app/controllers/application_controller.rb

```diff
@@ -37,4 +37,10 @@ def configure_permitted_parameters
       keys: %i[first_name last_name email password password_confirmation]
     )
   end
+
+  protected
+
+  def after_sign_in_path_for(_resource)
```

Comment:
You put `_resource` because of Rubocop, right? Whenever Rubocop complains about that it's because we're not using the argument but the caller is passing it. I prefer to use `def after_sign_in_path_for(*)` in this case.

The same logic applies for `def as_json(*)` instead of `def as_json(_options = {})` ;)

Reply:
Commenting here to start a new review, because I approved instead of requesting changes on my last comment :P